samples/powermax_revproxy_standalone_with_driver.yaml

```diff
@@ -79,8 +79,8 @@ spec:
     forceUpdate: false
     fsGroupPolicy: ReadWriteOnceWithFSType
     common:
-      # Image for CSI PowerMax driver v2.3.0
-      image: dellemc/csi-powermax:v2.3.0
+      # Image for CSI PowerMax driver v2.4.0
+      image: dellemc/csi-powermax:v2.4.0
```

Comment:
In the same file, we need to update the csi-reverseproxy image to v2.3.0 and any other places wherever its referenced

Reply:
updated the latest version .